op-challenger/game/fault/contracts/oracle.go

```diff
@@ -86,10 +86,14 @@ func (c *PreimageOracleContract) Addr() common.Address {
 }
 
 func (c *PreimageOracleContract) AddGlobalDataTx(data *types.PreimageOracleData) (txmgr.TxCandidate, error) {
-	call := c.contract.Call(methodLoadKeccak256PreimagePart, new(big.Int).SetUint64(uint64(data.OracleOffset)), data.GetPreimageWithoutSize())
+	call := c.AddGlobalDataTxCall(data)
 	return call.ToTxCandidate()
 }
 
+func (c *PreimageOracleContract) AddGlobalDataTxCall(data *types.PreimageOracleData) *batching.ContractCall {
```

Comment:
nit: I think typically this would be `CallAddGlobalData` - the `Tx` prefix generally indicates it returns a TxCandidate.  Although we don't normally return the `ContractCall` instance...